### DISCUSSION TITLE
Is jtop compatible?

### DISCUSSION THREAD

**Discussion by YoungjaeDev** [OE4T/meta-tegra#1037] [General]
Hello
Currently, I have configured the environment with meta-tegra bsp, but I don't know if my program I run uses cuda or hw-accelerate buf, decoder, so I want to use the monitoring program jtop, is it compatible?
Or is there a monitoring program supported by meta-tegra?
Thank you.

**Comment by ichergui** [comment]
Hello @youngjae-avikus 

Are you using `meta-tegra-community` repo ?  if so, we have the `jetson-stats` Yocto recipe [here](https://github.com/OE4T/meta-tegra-community/blob/master/recipes-devtools/python/python3-jetson-stats_3.1.2.bb)

**Comment by YoungjaeDev** [comment]
I'm using meta-tegra/dunfell
Is it impossible to use this repo?
What is the difference between `meta-tegra-community` and `dunfell`?

What does additional package mean?

Thank you

**Reply by dwalkes** [reply]
> I'm using meta-tegra/dunfell
> Is it impossible to use this repo?

No, you'd use `meta-tegra-community` as one of your layers.  Check out https://github.com/OE4T/tegra-demo-distro/tree/dunfell/layers for an example setup.  You might want to test drive https://github.com/OE4T/tegra-demo-distro/tree/dunfell as a working distribution and yocto setup if you haven't already.
